src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -46,6 +42,10 @@ class SettingsMenu extends Menu {
             menu.createItems(items, onItemSelect, itemOptions), 
             defaultItemIndex
         );
+        const categoryButton = menu.categoryButton && menu.categoryButton.element();
+        if (categoryButton && categoryButton.parentElement.firstChild === categoryButton) {
```

Comment:
Since `categoryButton.parentElement` is the same as `this.mainMenu.buttonContainer` can we make this a little shorter?

```suggestion
        if (this.mainMenu.buttonContainer.firstChild === categoryButton) {
```

Reply:
Mich cleaner, merged from the suggestion.

Reply:
Can't commit straight from suggestion (categoryButton undefined), but I've simplified the original check per the above.